apps/backend/user-service/src/test/resources/bootstrap-test.yaml

```
@@ -6,4 +6,4 @@ spring:
         enabled: false
 eureka:
   client:
-    enabled: false
+    enabled: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)